### PR TITLE
add serverspec tests and remove bindings

### PR DESCRIPTION
**Pull request by lmayorga1980**
@ferventcoder @jpogran,

Removed the bindings as mentioned in the last comment and also added some serverspec tests.

### CODE REVIEW

**Comment by ferventcoder** [comment]
@lmayorga1980 looks like you may want fetch the latest code and then rebase this against it. I'd prefer using the location where .fixtures.yml puts the dependencies.

**Comment by ferventcoder** [comment]
AppVeyor/TravisCI tests are green for master now, so when you rebase, keep them green. 👍

**Comment by lmayorga1980** [comment]
@ferventcoder , I noticed that the tests do not execute `vagrant` tests.  Do you want to keep the `Vagrantfile` and `serverspec` in the source code or prefer on extending the travis-ci feature that include these tests.

File: Puppetfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  :git => 'git@github.com:puppetlabs/puppetlabs-powershell.git',
+  :tag => '1.0.4'
+
+mod 'puppetlabs-iis',
+  :git => 'git@github.com:puppetlabs/puppetlabs-iis.git',
+  :tag => '4.1.0'
+
+mod 'puppet-windowsfeature',
+  :git => 'git@github.com:voxpupuli/puppet-windowsfeature.git',
+  :tag => 'v1.0.0'
```
Comment (ferventcoder) [inline Puppetfile:25]: This is no longer a dependency, please remove.

Comment (ferventcoder) [reply Puppetfile:25]: Or is it a transient dependency for something else?

**Comment by ferventcoder** [comment]
If there is a feature for travis-ci, it may not be a bad idea to get it working...

**Comment by lmayorga1980** [comment]
@ferventcoder , I will check it for a future MR. Let me know if it can be merged.

I guess is undertandable that they don't provide windows environment but I might be wrong.

https://docs.travis-ci.com/user/reference/overview/

File: Puppetfile (hunk 1 above)
Comment (lmayorga1980) [reply Puppetfile:25]: This is a transient dependency. I used the `Puppetfile` to bring down runtime dependencies so vagrant can use them.  Once we figure out the travis-ci feature I guess the whole Puppetfile will be gone as well as the Vagrantfile.

**Comment by lmayorga1980** [comment]
@ferventcoder can you check this PR?

**Comment by lmayorga1980** [comment]
Would be interested in having beaker support on this project? Something like this https://github.com/wazuh/ossec-puppet/pull/80

**Comment by jpogran** [comment]
Could the requestor or the owners of this repo untag me from this issue please? I no longer work with the company or products referenced here. Thanks in advance.

**Pull request closed without merging**